### PR TITLE
Feat: add deploy scripts

### PR DESCRIPTION
**Pull request by amusingaxl**

### CODE REVIEW

**Review by Farnoosh85 — CHANGES_REQUESTED** [review]
information on data protection

File: README.md [hunk 1]
```diff
@@ -36,18 +36,120 @@ bespoke spell in record time.
...
+| ``LITE-PSM-USDC-A` | BOTH | `TODO` |
+| ``LITE-PSM-USDC-A` | BUY  | `TODO` |
+| ``LITE-PSM-USDC-A` | SELL | `TODO` |
+-->
+
+#### `SingleOsmStopSpell`
+
+| Ilk         | Address                                      |
+| ----------- | -------------------------------------------- |
+| `ETH-A`     | `0x3a03F35ba1F5e015a288feD50cEB5342d4989E85` |
```
Comment (Farnoosh85) [inline README.md:133]: Organization validation

**Review by Farnoosh85 — CHANGES_REQUESTED** [review]

File: script/MultiClipBreakerDeploy.s.sol [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity ^0.8.16;
+
+import {Script} from "forge-std/Script.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {MCD, DssInstance} from "dss-test/MCD.sol";
```
Comment (oddaf) [inline script/MultiClipBreakerDeploy.s.sol:20]:
```suggestion
```

Unused imports

File: script/MultiLineWipeDeploy.s.sol [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity ^0.8.16;
+
+import {Script} from "forge-std/Script.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {MCD, DssInstance} from "dss-test/MCD.sol";
```
Comment (oddaf) [inline script/MultiLineWipeDeploy.s.sol:20]:
```suggestion
```
Unused imports

File: script/MultiOsmStopDeploy.s.sol [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity ^0.8.16;
+
+import {Script} from "forge-std/Script.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {MCD, DssInstance} from "dss-test/MCD.sol";
```
Comment (oddaf) [inline script/MultiOsmStopDeploy.s.sol:20]:
```suggestion
```

Unused imports

File: script/SingleClipBreakerDeploy.s.sol [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity ^0.8.16;
+
+import {Script} from "forge-std/Script.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {MCD, DssInstance} from "dss-test/MCD.sol";
```
Comment (oddaf) [inline script/SingleClipBreakerDeploy.s.sol:20]:
```suggestion
```
Unused imports

File: script/SingleDdmDisableDeploy.s.sol [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity ^0.8.16;
+
+import {Script} from "forge-std/Script.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {MCD, DssInstance} from "dss-test/MCD.sol";
```
Comment (oddaf) [inline script/SingleDdmDisableDeploy.s.sol:20]:
```suggestion
```
Unused imports

File: script/SingleLineWipeDeploy.s.sol [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity ^0.8.16;
+
+import {Script} from "forge-std/Script.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {MCD, DssInstance} from "dss-test/MCD.sol";
```
Comment (oddaf) [inline script/SingleLineWipeDeploy.s.sol:20]:
```suggestion
```
Unused imports

File: script/SingleLitePsmHaltDeploy.s.sol [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity ^0.8.16;
+
+import {Script} from "forge-std/Script.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {MCD, DssInstance} from "dss-test/MCD.sol";
```
Comment (oddaf) [inline script/SingleLitePsmHaltDeploy.s.sol:20]:
```suggestion
```
Unused imports

File: script/SingleOsmStopDeploy.s.sol [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity ^0.8.16;
+
+import {Script} from "forge-std/Script.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {MCD, DssInstance} from "dss-test/MCD.sol";
```
Comment (oddaf) [inline script/SingleOsmStopDeploy.s.sol:20]:
```suggestion
```
Unused imports

File: script/SplitterStopDeploy.s.sol [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity ^0.8.16;
+
+import {Script} from "forge-std/Script.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {MCD, DssInstance} from "dss-test/MCD.sol";
```
Comment (oddaf) [inline script/SplitterStopDeploy.s.sol:20]:
```suggestion
```
Unused imports

**Comment by oddaf** [comment]
All deployed contracts in the readme were:
a. Verified on Etherscan
b. Contain the correct collateral
c. Bytecode matches the version in the repository (commit f9308604f731ab180f6eb10d463b3edeb6a9a127, used in the Chainsecurity audit).

**Review by oddaf — APPROVED** [review]

**Review by 0xp3th1um — APPROVED** [review]
lgtm
The deployed contracts and their bytecode on Etherscan were also verified against commit https://github.com/makerdao/dss-emergency-spells/commit/f9308604f731ab180f6eb10d463b3edeb6a9a127 (Chainsecurity audit)

**Pull request merged**